test/io/test_io.py

```diff
@@ -964,45 +964,45 @@ def test_log_level(level, defaultenv):
             assert len(output) == 0
         elif level == "error":
             assert re.match(
-                r'- - - \[.+\] "GET / HTTP/1.1" 500 - "" "python-requests/.+"',
+                r'- - - \[.+\] "GET / HTTP/1.1" 500 \d+ "" "python-requests/.+"',
```

Comment:
This is a neat free addition. The logs also add the value of the `Content-Length` here (after the HTTP status).

Reply:
Super cool! Before it used to be 0 or blank?

Reply:
It used to be `-`, like `500 - - ""`. The modified docs have an example of this.